File: 1-two-sum/1-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        
        """
        hashMap = {}
        if len(nums) == 2: return [0,1]
        for index, num in enumerate(nums, start=0):
                RequiredVal = target - num
                if RequiredVal in hashMap:
                        return [hashMap[RequiredVal], index]
                hashMap[num] = index

File: 1-two-sum/1-two-sum.js

var twoSum = function(nums, target) {
    let hashMap = {};
    if (nums.length == 2){ return [0, 1]}
    for (let i = 0; i < nums.length; i++) {
        let RequiredVal = target - nums[i];
        if (RequiredVal in hashMap) {
            return [hashMap[RequiredVal], i];
        }
        hashMap[nums[i]] = i;

    }
};